airbyte-integrations/connectors/source-mssql/src/main/resources/spec.json

```diff
@@ -100,7 +100,7 @@
           }
         ]
       },
-      "replication": {
+      "replication_method": {
```

Comment:
Nitpick. MS SQL uses the `replication_type` field under `replication_method` to distinguish between different replication types. However, the corresponding field in MySQL and Postgres is `method`. Should we also unify this field and replace `replication_type` with `method` for MS SQL?

Reply:
> Nitpick. MS SQL uses the `replication_type` field under `replication_method` to distinguish between different replication types. However, the corresponding field in MySQL and Postgres is `method`. Should we also unify this field and replace `replication_type` with `method` for MS SQL?

 replaced replication_type with method for MS SQL